src/app/contextBroker/contextBroker.cpp

```diff
@@ -1182,6 +1182,13 @@ void orionExit(int code, const std::string& reason)
 */
```

Comment:
Maybe just a sentence like "Fix: several memory leaks has been fixed" is enough? Or are you able to be more specific about which old leaks has been fixed?